lib/card_connect_gateway/authorization/response.rb

```diff
@@ -39,6 +39,177 @@ class Response < BaseResponse
       Y = 'Y'
       Z = 'Z'
 
+      VISA_ADDRESS_MATCH_CODES = [
+        A, # Address matches, ZIP does not.
+        B, # Street addresses match. 
+        D, # Street addresses and postal codes match. 
+        F, # Street addresses and postal codes match. UK Only
+        M, # Street address and postal code match. 
+        X, # Not applicable. If present, replaced with ”Y” by V.I.P. Available for U.S. issuers only.
+        Y  # Street address and postal code match.
+      ]
+      VISA_ZIPCODE_MATCH_CODES = [
+        D, # Street addresses and postal codes match. 
+        F, # Street addresses and postal codes match. UK Only
+        M, # Street address and postal code match.
+        P, # Postal code match.
+        W, # Not applicable. If present, replaced with “Z” by V.I.P. Available to U.S. issuers only
+        X, # Not applicable. If present, replaced with ”Y” by V.I.P. Available for U.S. issuers only.
+        Y, # Street address and postal code match.
+        Z  # Postal/ZIP matches; street address does not match or Street address not included in request.
+      ]
+
+      VISA_ADDRESS_MISMATCH_CODES = [
+        C, # Street address and postal code not verified due to incompatible formats.
+        N, # No match. Acquirer sent postal/ZIP code only, or, street address only, or, both postal code and street address. Also used when acquirer requests AVS, but sends no AVS data in Field 123.
+        Z  # Postal/ZIP matches; street address does not match or Street address not included in request.
+      ]
+
+      VISA_ZIPCODE_MISMATCH_CODES = [
+        A, # Address matches, ZIP does not.
+        B, # Postal code not verified due to incompatible formats.
```

Comment:
Document says this is 
```Street addresses match. Postal code not verified due to incompatible
formats.``` 
Don't know if the street addresses match part is important though.